### DISCUSSION TITLE
Packages API does not support pagination

### DISCUSSION THREAD

**Discussion by olehm-ma** [community/community#40255] [Actions]
### Select Topic Area

Bug

### Body

Using the method https://docs.github.com/en/rest/packages#list-packages-for-an-organization I get only the first page (30 items). I cannot iterate over all existing packages for my organisation.

**Comment by olehm-ma** [comment]
Looks like it works with page parameter, however it is not mentioned in the documentation.

**Comment by ossanna16** [comment]
Hi there @olehm-ma and welcome to our community! Thank you for reporting this issue!
